### PR TITLE
Profile page created

### PR DESCRIPTION
**Pull request by Rojesh04Mote**

### CODE REVIEW

File: src/components/elements/CustomSessionPopup/index.js [hunk 1]
```diff
@@ -1,84 +1,70 @@
-import React from 'react'
-import { View, Text, TouchableOpacity } from 'react-native'
```
Comment (SushantTAN) [inline src/components/elements/CustomSessionPopup/index.js:2]: one space after this import

File: src/components/elements/CustomSessionPopup/index.js [hunk 2]
```diff
@@ -1,84 +1,70 @@
-import React from 'react'
```
Comment (SushantTAN) [inline src/components/elements/CustomSessionPopup/index.js:1]: comments required in the top of the file.

File: src/components/elements/CustomSessionPopup/index.js (hunk 1 above)
Comment (Rojesh04Mote) [reply src/components/elements/CustomSessionPopup/index.js:2]: yo kundan ko ho

File: src/components/elements/CustomSessionPopup/index.js (hunk 2 above)
Comment (Rojesh04Mote) [reply src/components/elements/CustomSessionPopup/index.js:1]: yo ni kundan ko ho

File: src/components/elements/CustomButton/index.js [hunk 3]
```diff
@@ -7,7 +7,8 @@
  */
 
 import React from 'react';
-import { View, Text, TouchableOpacity } from 'react-native';
+
```
Comment (SushantTAN) [inline src/components/elements/CustomButton/index.js:10]: this line is not required

**Review by SushantTAN — APPROVED** [review]

**Pull request merged**